backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -153,20 +153,80 @@ public Person(
     this.testResultDeliveryPreference = testResultDeliveryPreference;
   }
 
-  public Person(PersonName names, Organization org, Facility fac) {
+  public Person(
+      Organization organization,
+      Facility facility,
+      String lookupId,
+      String firstName,
+      String middleName,
+      String lastName,
+      String suffix,
+      LocalDate birthDate,
+      StreetAddress address,
+      String country,
+      PersonRole role,
+      List<String> emails,
+      String race,
+      String ethnicity,
+      List<String> tribalAffiliation,
+      String gender,
+      Boolean residentCongregateSetting,
+      Boolean employedInHealthcare,
+      String preferredLanguage,
+      TestResultDeliveryPreference testResultDeliveryPreference) {
+    this(
+        organization,
+        lookupId,
+        firstName,
+        middleName,
+        lastName,
+        suffix,
+        birthDate,
+        address,
+        country,
+        role,
+        emails,
+        race,
+        ethnicity,
+        tribalAffiliation,
+        gender,
+        residentCongregateSetting,
+        employedInHealthcare,
+        preferredLanguage,
+        testResultDeliveryPreference);
+    this.facility = facility;
+  }
+
+  public Person(PersonName names, Organization org, Facility facility) {
     super(org);
-    this.facility = fac;
+    this.facility = facility;
     this.nameInfo = names;
     this.role = PersonRole.STAFF;
   }
 
-  public Person(PersonName names, Organization org, Facility fac, PersonRole role) {
+  public Person(PersonName names, Organization org, Facility facility, PersonRole role) {
     super(org);
-    this.facility = fac;
+    this.facility = facility;
     this.nameInfo = names;
     this.role = role;
   }
 
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+    Person person = (Person) o;
+    return nameInfo.equals(person.nameInfo)
```

Comment:
One other thing we may want to check for is `isArchived` - it doesn't matter as much for this use case since there's no way a patient from the CSV would be archived before the duplicate patient is loaded, but long-term I think it's reasonable to consider archived and active patients as two different entities.

Reply:
That's fair! Do you want to add that distinction now?

Reply:
Yes please!

Reply:
✅